lib/pages/people-page.js

```diff
@@ -57,18 +57,38 @@ export default class PeoplePage extends AsyncBaseContainer {
 	}
 
 	async viewerDisplayed( username ) {
-		return DriverHelper.isEventuallyPresentAndDisplayed(
-			this.driver,
-			By.css( `.people-profile__login[data-e2e-login="${ username }"]` )
+		await DriverHelper.isEventuallyPresentAndDisplayed( this.driver, By.css( '.count' ) );
+		let noOfViewers = parseInt(
+			await this.driver.findElement( By.css( '.section-header__label .count' ) ).getText()
 		);
+
+		let retries = Math.round( noOfViewers / 10 );
+		let displayed;
+
+		for ( let i = 0; i <= retries; i++ ) {
+			displayed = await DriverHelper.isEventuallyPresentAndDisplayed(
+				this.driver,
+				By.css( `.people-profile__login[data-e2e-login="${ username }"]` ),
+				500
+			);
+			if ( displayed ) {
+				break;
+			} else {
+				await DriverHelper.scrollToBottom( this.driver, By.css( '.layout__primary' ) );
```

Comment:
This `.layout__primary` selector solved scrolling issue. At first, I used `.people-selector__infinite-list` and it worked only locally.